docs/flatcar-linux/aws.md

```diff
@@ -208,7 +208,7 @@ Check the [variables.tf](https://github.com/kinvolk/lokomotive-kubernetes/blob/m
 | dns_zone | AWS Route53 DNS zone | "aws.example.com" |
 | dns_zone_id | AWS Route53 DNS zone id | "Z3PAABBCFAKEC0" |
 | ssh_authorized_key | SSH public key for user 'core' | "ssh-rsa AAAAB3NZ..." |
-| asset_dir | Path to a directory where generated assets should be placed (contains secrets, used to destroy the cluster), cannot be a relative path | "/home/user/.secrets/clusters/tempest" |
+| asset_dir | Path to a directory where generated assets should be placed (contains secrets, used to destroy the cluster), cannot be a relative path | "./assets" |
```

Comment:
It says "cannot be a relative path" :joy: And AWS is the only one that says this?

Maybe this was because of a limitation in other tools using this that are now fixed? :thinking: I'm quite sure it is not a problem in using the terraform module with a relative path, though. So I think it is safe to remove this? Maybe we should double check this, my memory is not the best :)

Reply:
removed it.

Testing it in a bit.

Reply:
Thanks! Maybe the CI uses a relative path for tests or we can change that? In that way, you don't have to test :)

Reply:
Yes, this good idea :-)